src/main/java/lotto/data/Lotto.java

```diff
@@ -0,0 +1,32 @@
+package lotto.data;
+
+import java.util.Collections;
+import java.util.List;
+
+import static lotto.service.LottoGame.makeLottoNumberPool;
+
+public class Lotto {
+    private List<Integer> numbers;
+
+    public Lotto() {
+        this.numbers = makeNumbers();
```

Comment:
```suggestion
        this(makeNumbers());
```
주생성자를 이용하면 초기화 로직을 재사용가능하겠네요!

Reply:
안녕하세요! 시간내주시고 리뷰해주셔서 감사합니다 :) 
제안해주신것처럼 하려고 하니, "Cannot reference 'Lotto.makeNumbers' before supertype constructor has been called" 에러로 활용하지 못하는것 같았습니다.. ㅠㅠ

Reply:
`makeNumbers`가 `static`이 아니어서 안되는군요! 😂